indra/newview/llgltfmateriallist.h

```diff
@@ -74,6 +74,8 @@ class LLGLTFMaterialList
     // Automatically called once per frame, but may be called explicitly
     // for cases that care about the done_callback forwarded to LLCoros::instance().launch
     static void flushUpdates(void(*done_callback)(bool) = nullptr);
+
+    static void addUpdateCallback(void(*update_callback)(const LLUUID& object_id, S32 side));
```

Comment:
This only works for objects with seelctions, so might be better to name it addSelectionUpdateCallback or in some other way that indicates that this isn't a blanket update.